daemon/response.go

```diff
@@ -27,8 +27,11 @@ import (
 	"path/filepath"
 	"strconv"
 
+	"github.com/gorilla/websocket"
```

Comment:
Do we need to have golang-github-gorilla-websocket in debian/control somewhere or is golang-websocket-dev enough?